base/operators.jl

```diff
@@ -328,9 +328,31 @@ eltype(::Type{Any}) = Any
 eltype(t::DataType) = eltype(supertype(t))
 eltype(x) = eltype(typeof(x))
 
-# function pipelining
+# function pipelining, composition & negation
+
 |>(x, f) = f(x)
 
+"""
+    ∘(f, g)
+    f ∘ g
```

Comment:
I added `.. function:: ∘(f, g)` to base.rst and, although it worked for `.. function:: !(f::Function)`, I get the following:
```
WARNING: missing docs for signature:

    ∘(f, g)
```
Is this something to do with unicode operators?